Secureboot: No rule to make target 'cert/sign.pub'

**What happened**:
```
[Warning] one or more build args were not consumed: [VERSION]
e7448241692e748cbb9c13fb388e9cce7b3e544bfca6d83fb07413fb0539ca16
make[1]: Leaving directory '/home/vagrant/gardenlinux/container'
make: *** No rule to make target 'cert/sign.pub', needed by 'metal-secureboot-dev'.  Stop.
```

You can work around by creating any other image before.

**What you expected to happen**:
This should work; therefore the dependency handling within the `Makefile` needs to be updated.

**How to reproduce it (as minimally and precisely as possible)**:
Make sure to have a clean GardenLinux GIT repo without having any other image built. Afterwards run `make metal-secureboot-dev`.

--- a/Makefile
+++ b/Makefile
@@ -162,10 +162,10 @@ metalk: build-environment $(SECUREBOOT_CRT)
 metalk-dev: build-environment $(SECUREBOOT_CRT)
 	./build.sh $(BUILD_OPTS) --skip-build --features metal,khost,_pxe,_dev $(BUILDDIR) $(VERSION)
 
-metal-secureboot: container-build cert/sign.pub
+metal-secureboot: build-environment $(SECUREBOOT_CRT) cert/sign.pub
 	./build.sh $(BUILD_OPTS) --skip-build --features server,metal,_secureboot $(BUILDDIR) $(VERSION)
 
-metal-secureboot-dev: container-build cert/sign.pub
+metal-secureboot-dev: build-environment $(SECUREBOOT_CRT) cert/sign.pub
 	./build.sh $(BUILD_OPTS) --skip-build --features server,metal,_secureboot,_dev $(BUILDDIR) $(VERSION)
 
 clean:

